### PR TITLE
Update MIGRATION.md

### PR DESCRIPTION
**Pull request by mcm1957**
npm uninstall eslint-plugin-security should be removed too

### CODE REVIEW

**Comment by foxriver76** [comment]
Never seen this plugin, I don't think it is very common. Before we add every edgy package, maybe we should leave a sentence to explain to uninstall all eslint plugins which are not afterwards added again to the new config. I mean someone could use this plugin again with the config if he wants to? It also works with flat config .. https://www.npmjs.com/package/eslint-plugin-security

**Comment by mcm1957** [comment]
OK, I agree that there could be some additional rule files.

I do not know where / when eslint-plugin-securita was invented but it was part of a discussion at Telegram and I no stubled at it when migrating srm adapter to community.

So I added an extra paragraph. Please read and feel free to adapt. I would like to guide those users who do not know what this plgin is good for to remiove it while ist comüpletly ok to leave / add ist for those who want to use these checks.

File: MIGRATION.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ The steps are valid for vanilla Javascript and TypeScript repositories. If you a
...
 
+- [ ] remove all other eslint / prettier packages you do not plan to add to you personal configuration later
+
+  You should remove all `eslint-*` and `prettier.*` packages except those you want to add to the adapterspecific configuration file later. The standard configuration `@iobroker/eslint-config` does not require any additional packages installed at repository level. So in most cases you can safely remove all `eslint-*` and `prettier-*` packages.  
+
+  To remove i.e. `eslint-plugin-security` issue this command:
+      
+      npm uninstall eslint-plugin-security
+
+  Please note that any additional packages you want to install must be verified to work with eslint 9 and must be added to your configuration file manually .
```
Comment (foxriver76) [inline MIGRATION.md:28]:
```suggestion
  Please note that any additional packages you want to install must be verified to work with ESLint 9 and must be added to your configuration file manually .
```

**Review by foxriver76 — APPROVED** [review]
lgtm just a small "typo"

**Comment by mcm1957** [comment]
Thanks

**Pull request merged**